.gitignore

```diff
@@ -45,8 +45,6 @@ etc/kolla
 
 # Ignore wazuh
 etc/kayobe/ansible/roles/wazuh-ansible/
-etc/kayobe/ansible/wazuh/certificates/*
```

Comment:
This was annoying as you can forget to check in the certificates, so I've not add rules for the new paths